src/styles/table/table.css

```diff
@@ -0,0 +1,40 @@
+@import url(../../themes/themes.css);
+/*
+ *  Styling for native-HTML tables.
+ **/
+
+table {
+  border-collapse: collapse;
+}
+
+tr {
+  background-color: var(--surface);
+  border-bottom: 1px solid var(--surface-2dp);
+}
+
+tr:hover {
+  background-color: var(--surface-1dp);
+}
+
+tr:hover tr:hover {
+  background-color: var(--surface-2dp);
+}
+
+td,th {
+  color: var(--on-surface);
+  padding: 10px;
+}
+
+td {
+  font-size: 0.813em;
```

Comment:
Set font-size using percents, here and below:

font-size: 90%;

Reply:
Done